kriya-tech/src/App.jsx

```diff
@@ -14,6 +14,7 @@ const App = () => {
       <Section />
       <Cards />
       <Footer />
+      <ContactPage />
```

Comment:
Please add it through Navbar instead of adding directly  below footer.

We can add few interactive buttons inside footer though which redirects to Contact page.